### ISSUE TITLE
PDF template path changed after update 3.2

### ISSUE THREAD

**Issue by kevinmu17** [craftcms/commerce#1887]
**Description**
When updating to Craft Commerce 3.2 it comes with the new ability to select PDF's. Somehow it's overwriting the "PDF Template Path" of existing configurations to "_shop/special/receipt_"

This broke my webshop

**To Reproduce**
How, and what happened?
Update commerce from an existing setup to 3.2+
2 ways to check
1. Order something with an email being send and the PDF attached to it (this will break)
2. Download the PDF from the order within the CP (download pdf button)

**Expected behavior**
Leave my configuration alone :)

**Additional info**
- Craft version: 3.5.15.1
- Commerce version: 3.2.12
- PHP version: 7.3.22
- Database driver & version: MariaDB 10.3
- Other Plugins & versions: N/A

**Comment by lukeholder** [comment]
It should have migrated your existing settings:

https://github.com/craftcms/commerce/blob/develop/src/migrations/m200801_233755_pdfs.php#L83

This is the first time someone has reported this issue. Guessing you have since updated the new PDF settings with the correct path and everything is working again?

Interested to know if you were doing anything special to override that general setting (`orderPdfPath`)?

**Comment by kevinmu17** [comment]
> Guessing you have since updated the new PDF settings with the correct path and everything is working again?

Correct, everything worked fine again after I changed it back. FYI: I came from craft v3.0.6

> Interested to know if you were doing anything special to override that general setting (orderPdfPath)?

No, when launched it was set to /shop/receipt and after the update it added the Default (handle: order) and made this default with template path /shop/special/receipt. But it changed/overwrite the existing config as well with /shop/special/receipt.

configurations are made in _Commerce > system settings > PDF's_ no overwrites from general configs or something else.
